src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Logging;
+using Dynamo.Models;
+using Dynamo.PythonMigration.Properties;
+using Dynamo.UI.Prompts;
+using Dynamo.ViewModels;
+using Dynamo.Wpf.Extensions;
+using PythonNodeModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+
+namespace Dynamo.PythonMigration
+{
+    public class PythonMigrationViewExtension : IViewExtension
+    {
+        private const string EXTENSION_NAME = "Workspace References";
+        private const string EXTENSION_GUID = "1f8146d0-58b1-4b3c-82b7-34a3fab5ac5d";
+        private const string NOTIFICATION_TITLE = "IronPythonNotification";
```

Comment:
maybe we make this more generic ? since other things are likely to be added to the migration extension
could remove this and use the extension name actually.

```suggestion
        private const string NOTIFICATION_TITLE = "Python Migration";
```